src/index.ts

```diff
@@ -0,0 +1,27 @@
+import "reflect-metadata";
+import { createConnection } from "typeorm";
+import * as express from "express";
+import * as bodyParser from "body-parser";
+import * as helmet from "helmet";
+import * as cors from "cors";
+import routes from "./routes";
+
+//Connects to the Database -> then starts the express
+createConnection()
+  .then(async connection => {
+    // Create a new express application instance
+    const app = express();
+
+    // Call midlewares
+    app.use(cors());
+    app.use(helmet());
+    app.use(bodyParser.json());
+
+    //Set all routes from routes folder
+    app.use("/", routes);
+
+    app.listen(3000, () => {
+      console.log("Server started on port 3000!");
```

Comment:
har du ikke super meget lyst til at gøre at den her logger med bunyun i stedet for god logging?